.github/codecov.yml

```diff
@@ -1,9 +1,6 @@
 codecov:
   require_ci_to_pass: yes
 
-ignore:
-  - internal/examples/proto/gen
```

Comment:
Ok to scope creep that in a Go version bump? :)

Reply:
:see_no_evil: